### PR TITLE
adding nodezoo-updater, dequeue -> Do not merge until:

### PR DESCRIPTION
**Pull request by Georgette**
Depends on 

[nodezoo-dequeue PR 4](https://github.com/nodezoo/nodezoo-dequeue/pull/4)
nodezoo-dequeue npm publish (if the version isn't 1.0.0, I need to change package.json)

https://github.com/nodezoo/nodezoo-npm-update/issues/9
https://github.com/nodezoo/nodezoo-npm-update/issues/11
nodezoo-updater publish to npm (if the version isn't 1.0.0, I need to change package.json)

Since I have this locally, I wanted to push a PR on it for later.

### CODE REVIEW

File: fuge/system.yml [hunk 1]
```diff
@@ -52,6 +52,18 @@ nodezoo_travis:
...
+  container_name: nodezoo_updater
+  env_file: ../system.env
+  net: host
+
+nodezoo_dequeue:
+  build: ../node_modules/nodezoo-dequeue/
+  container_name: nodezoo_dequeue
+  env_file: ../system.env
+  net: host
+
```
Comment (Georgette) [inline fuge/system.yml]: I did not submit a restart setting @AdrianRossouw what should it be here?

File: fuge/system.yml [hunk 2]
```diff
@@ -52,6 +52,18 @@ nodezoo_travis:
   net: host
   restart: always
 
+nodezoo_updater:
```
Comment (Georgette) [inline fuge/system.yml:55]: I did not submit a restart setting for updater, or dequeue. @AdrianRossouw what should this be?

Comment (mcdonnelldean) [inline fuge/system.yml:55]: How do you mean?

**Comment by mcdonnelldean** [comment]
Lets double check this one when you are on @Georgette

**Pull request merged**